edisyn/synth/novationastation/NovationAStation.java

```diff
@@ -305,24 +294,74 @@ public JFrame sprout()
         return super.sprout();
     }
 
+    @Override
+    public boolean testVerify(Synth synth2, String key, Object obj1, Object obj2)
+    {
+        return UNMAPPED_KEYS.contains(key);
+    }
+
+    ////
+    // librarian support
+    ////
+    @Override
+    public String[] getBankNames()
+    {
+        return BANKS;
+    }
+
+    @Override
+    public String[] getPatchNumberNames()
+    {
+        return PATCHES;
+    }
+
+    @Override
+    public boolean getSupportsPatchWrites()
+    {
+        return true;
+    }
+
     @Override
     public boolean librarianTested()
     {
-        // Override this method to return true to indicate that the librarian for this
-        // editor has been tested reasonably well and no longer requires a warning to the
-        // musician when he attempts to use it. By default this method returns false.
-        return false; 
+        return true;
```

Comment:
not there yet, still found an errorcase (sending multipe patches to AStation), needs investigation.

Reply:
that was a race condition, bypaased for now by adding  some extra delay
*ref [next commit](https://github.com/eclab/edisyn/pull/84/commits/a06375a1995cb443fb40f8d81e9164e48ebc0c3f)